### PR TITLE
Better error handling in get-daml.sh

### LINKED ISSUE #1034: get-daml.sh proper error reporting.
(via Tristan Fairbarn)

get-daml.sh reports "Successfully installed DAML" even though it was not successful.

```
$ curl https://get.daml.com/ -sSf | sh
Determining latest DAML version...
Latest DAML version is 0.12.16.
Downloading latest DAML SDK. This may take a while.
##                                                                         2.8%
curl: (56) OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 54
Extracting SDK release tarball.
gzip: (stdin): unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
/var/folders/x2/qmb_1zmx1z1_037ljpzjz0hh0000gn/T/tmp.CXu8UxzS/sdk/install.sh: line 5: /var/folders/x2/qmb_1zmx1z1_037ljpzjz0hh0000gn/T/tmp.CXu8UxzS/sdk/daml/daml: No such file or directory
Successfully installed DAML.
```

### PR DESCRIPTION
**Pull request by associahedron**
Fixes #1034. Tested on Linux.

Also adds a bit more information about DAML in the comment at the top per Neil's request.

### CODE REVIEW

File: daml-assistant/get-daml.sh [hunk 1]
```diff
@@ -4,21 +4,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
 #
+# DAML is an open-source privace aware smart contract language.
```
Comment (cocreature) [inline daml-assistant/get-daml.sh]:
```suggestion
# DAML is an open-source privacy-aware smart contract language.
```

File: daml-assistant/get-daml.sh [hunk 2]
```diff
@@ -4,21 +4,24 @@
...
 # This script downloads and installs the DAML SDK on Linux and macOS.
 # This will overwrite any existing installation in ~/.daml
+# For more information please visit daml.com and docs.daml.com
```
Comment (neil-da) [inline daml-assistant/get-daml.sh]: https://daml.com etc - that way it's got a chance of rendering as a hyperlink if viewed on the console or a text editor.

Comment (associahedron) [reply daml-assistant/get-daml.sh]: :+1:

File: daml-assistant/get-daml.sh (hunk 1 above)
Comment (associahedron) [reply daml-assistant/get-daml.sh]: Thanks!

**Pull request merged**